.github/workflows/release.yaml

```diff
@@ -17,25 +16,27 @@ jobs:
       - integration-test
     env:
       CHARMCRAFT_AUTH: ${{ secrets.CHARMHUB_TOKEN }}
+    strategy:
+      matrix:
+        bundle-name: [postgresql-k8s-bundle]
```

Comment:
We can probably remove the matrix now as there should be only one bundle (with TLS enabled).